### PR TITLE
Create directive for uploading images on front end

### PR DESCRIPTION
**Pull request by jake-shasteen**

### CODE REVIEW

File: client/javascript/app/home/home.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+  <d-upload-image-form></d-upload-image-form>
```
Comment (MrMarkMaker) [inline client/javascript/app/home/home.html]: Sweet.  I used this, uploaded images successfully. Now that I've been able to use it, can the directive be removed from <home>?

File: server/routes/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const conventionController = require( '../controllers/conventionController' );
 const uploadController = require( '../controllers/uploadController' );
 const helpers = require( '../lib/helpers' );
 const multer = require( 'multer' );
-const upload = multer({ dest: 'uploads/', limits: { fileSize: 3000000 } });
+const upload = multer({ dest: 'client/uploads/', limits: { fileSize: 3000000 } });
```
Comment (MrMarkMaker) [inline server/routes/index.js]: There is a file folder in Geekmapper that is 'uploads' as well as 'client/uploads'.  I understand images will upload to 'client/uploads' but can we delete the 'geekmapper/uploads' folder with this commit or is it meant for something in the future?

File: client/javascript/app/home/home.html (hunk 1 above)
Comment (jake-shasteen) [inline client/javascript/app/home/home.html]: fixed

File: server/routes/index.js (hunk 2 above)
Comment (jake-shasteen) [inline server/routes/index.js]: the uploads folder is not tracked by git, so you can safely delete geekmapper/uploads.

**Pull request merged**